frontend/src/app/testResults/TestResultCorrectionModal.tsx

```diff
@@ -155,7 +161,9 @@ export const DetachedTestResultCorrectionModal = ({
         showNotification(alert);
       })
       .finally(() => {
-        closeModal();
+        setTimeout(() => {
```

Comment:
The `setTimeout` is used here to allow time for the notification to show up before rerouting. might be a bug in the notification system.